src/content/docs/browser/browser-monitoring/installation/install-browser-monitoring-agent.mdx

```diff
@@ -22,128 +22,125 @@ If you don't have one already, [create a New Relic account](https://newrelic.com
 
 ## Enable browser monitoring [#enable]
 
-To enable browser: Go to **[one.newrelic.com](https://one.newrelic.com)**, select **Browser**, and then select **Add more data**. The UI gives you instructions. 
+Browser Pro+SPA is the default agent when you enable browser monitoring. This automatically gives you access to all of our browser monitoring features.
```

Comment:
I don't know what Browser Pro+SPA is... Do we need to clarify what this is?

Reply:
Or maybe just point to the table at the bottom of the doc?

Reply:
Good catch! We are trying to avoid plinko in that doc. I probably took it too far. I'll clarify.